RobotCode/src/main/java/frc/robot/commands/ToggleIntakeCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class ToggleIntakeCommand extends Command
+{
+
+	private final IntakeSubsystem Intake;
+
+	public ToggleIntakeCommand(IntakeSubsystem intake)
+	{
+		this.Intake = intake;
+	}
+
+	@Override
+	public void execute()
```

Comment:
`execute` runs 20 times per second. You should change to this `initialize`, which runs once when the command is started.